package.json

```diff
@@ -1,23 +1,28 @@
 {
   "dependencies": {
-    "bun-types": "latest",
     "express": "^4.18.2",
     "mitata": "^0.1.3",
     "peechy": "0.4.32",
     "react": "next",
-    "react-dom": "next"
+    "react-dom": "next",
+    "svelte": "^3.52.0"
```

Comment:
i don't remember why svelte is here

Reply:
It's used by `bun-loader-svelte.ts` which is imported in `plugins.test.ts`. I think it makes sense for all the dependencies needed for tests to live in the root `package.json` but we could add a separate `package.json` in `tests/bun.js` if you prefer